### ISSUE TITLE
[BUG] CSV Export of results converts 0 integer to empty field

### ISSUE THREAD

**Issue by roger-duthie-pivotal** [cweijan/vscode-mysql#1056]
- Database type and version: Postgres 14.9
- Client version 7.0.6

When exporting the results of a query toa CSV, an integer zero (0) is exported as an empty field (,,)

e.g. 
![image](https://github.com/cweijan/vscode-database-client/assets/139114058/46c9c3a1-3d4a-486f-8089-ba51a234cc36)

turns into
```csv
audio_file_id,audio_file_url,roi_id,roi_start,roi_end,model_label
104775,[...],94687,,3,[...]
```

**Comment by cweijan** [comment]
Thanks for the feedback, I will fix it in the next version.

**Comment by roger-duthie-pivotal** [comment]
Some additional details that I just discovered: this doesn't happen if the field is a varchar: 

i.e.
```sql
cast(roi.start as varchar) as "roi_start",
```
... means that the CSV is exported with zeroes where the field was a zero.

**Comment by cweijan** [comment]
Fixed in version 7.0.7.
